src/generateBreaks/normalizeClassBreaks.js

```diff
@@ -1,12 +1,19 @@
 'use strict'
+const _ = require('lodash')
+
 function getFieldValues (features, field) {
-  return features.map((feature, index) => {
+  if (!(field in features[0].properties)) throw new Error('Cannot use values from unrecognized field: ' + field)
```

Comment:
this error message may be misleading. `classification field missing from first feature` is more accurate here